docker-compose.yml

```
@@ -61,47 +61,46 @@ services:
       - ./src:/app/src
     init: true
     environment:
-      - USE_DEVELOPMENT_S3=1
-      - APP_PATH=${APP_PATH:-/index.html}
-      - AUTH_METHOD_OVERRIDE=${AUTH_METHOD_OVERRIDE:-}
-      - CLOUDFRONT_KEY_PAIR_ID=${CLOUDFRONT_KEY_PAIR_ID}
-      - CLOUDFRONT_PRIVATE_KEY=${CLOUDFRONT_PRIVATE_KEY}
-      - DEBUG=${DEBUG:-}
-      - DEV_NETWORK_IP=${DEV_NETWORK_IP:-localhost}
-      - DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL=${DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL}
-      - DEFAULT_FROM_EMAIL=${DEFAULT_FROM_EMAIL}
-      - DEFAULT_IDP_DOMAIN=${DEFAULT_IDP_DOMAIN}
-      - IPV4_INTERNAL_NETWORK_REGEX=${IPV4_INTERNAL_NETWORK_REGEX}
-      - IPV6_INTERNAL_NETWORK_REGEX=${IPV6_INTERNAL_NETWORK_REGEX}
-      - IS_DEV=${IS_DEV:-}
-      - IS_STAGING=${IS_STAGING}
-      - LOGIN_TEST_CODE=${LOGIN_TEST_CODE:-}
-      - LOGIN_TEST_EMAIL=${LOGIN_TEST_EMAIL:-}
-      - LOG_LEVEL=${LOG_LEVEL}
-      - MARKETING_DOMAIN=${MARKETING_DOMAIN}
-      - OVERRIDE_DATABASE_NAME=${OVERRIDE_DATABASE_NAME:-}
-      - OVERRIDE_DATABASE_HOSTNAME=${OVERRIDE_DATABASE_HOSTNAME:-}
-      - OVERRIDE_DATABASE_PASSWORD=${OVERRIDE_DATABASE_PASSWORD:-}
-      - OVERRIDE_DATABASE_PORT=${OVERRIDE_DATABASE_PORT:-}
-      - OVERRIDE_DATABASE_USERNAME=${OVERRIDE_DATABASE_USERNAME:-}
-      - PORT=${PORT:-8080}
-      - DATABASE_NAME=${DATABASE_NAME}
-      - DATABASE_HOSTNAME=${DATABASE_HOSTNAME}
-      - DATABASE_PASSWORD=${DATABASE_PASSWORD}
-      - DATABASE_PORT=${DATABASE_PORT}
-      - DATABASE_USERNAME=${DATABASE_USERNAME}
-      - QUOTES_DOMAIN=${QUOTES_DOMAIN}
-      - REQUIRE_HTTPS=${REQUIRE_HTTPS:-}
-      - S3_ACCESS_KEY_ID=${S3_ACCESS_KEY_ID}
-      - S3_BUCKET=${S3_BUCKET}
-      - S3_ENDPOINT=${S3_ENDPOINT}
-      - S3_SECRET_ACCESS_KEY=${S3_SECRET_ACCESS_KEY}
-      - SES_ACCESS_KEY_ID=${SES_ACCESS_KEY_ID}
-      - SES_AWS_REGION=${SES_AWS_REGION}
-      - SES_SECRET_ACCESS_KEY=${SES_SECRET_ACCESS_KEY}
-      - SESSION_SECRET=${SESSION_SECRET:-secret}
-      - SUPPORT_EMAIL=${SUPPORT_EMAIL}
-      - WHITELISTED_EMAILS=${WHITELISTED_EMAILS}
+      APP_PATH: ${APP_PATH:-/index.html}
+      AUTH_METHOD_OVERRIDE: ${AUTH_METHOD_OVERRIDE:-}
+      CLOUDFRONT_KEY_PAIR_ID: ${CLOUDFRONT_KEY_PAIR_ID:-}
+      CLOUDFRONT_PRIVATE_KEY: ${CLOUDFRONT_PRIVATE_KEY:-}
+      DATABASE_HOSTNAME: mysql
+      DATABASE_NAME: ${DATABASE_NAME:-ToadReader}
+      DATABASE_PASSWORD: ${DATABASE_PASSWORD}
+      DATABASE_PORT: ${DATABASE_PORT:-3306}
+      DATABASE_USERNAME: ${DATABASE_USERNAME}
+      DEBUG: ${DEBUG:-}
+      DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL: ${DEFAULT_DOMAIN_TO_QUERY_FOR_EMAIL}
+      DEFAULT_FROM_EMAIL: ${DEFAULT_FROM_EMAIL}
+      DEFAULT_IDP_DOMAIN: ${DEFAULT_IDP_DOMAIN:-}
+      DEV_NETWORK_IP: ${DEV_NETWORK_IP:-localhost}
+      IPV4_INTERNAL_NETWORK_REGEX: ${IPV4_INTERNAL_NETWORK_REGEX:-}
+      IPV6_INTERNAL_NETWORK_REGEX: ${IPV6_INTERNAL_NETWORK_REGEX:-}
+      IS_DEV: ${IS_DEV-1}
```

I would prefer this remains blank by default as we currently want it switched off in development, and it seems strange to have to specify an empty env variable to achieve this.
```suggestion
      IS_DEV: ${IS_DEV:-}
```
(either way, you are missing a colon)

---

Its not missing a colon, it is [defaulting with an empty replacement](https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#interpolation-syntax). However we can leave it as empty by default

---

OK thanks. In the future I hope we can remove `IS_DEV` as it causes confusing behaviour and we are not really using it any more.